extensions/arc/package.nls.json

```diff
@@ -83,47 +83,23 @@
-	"arc.postgres.server.group.cpu.min": "Min CPU cores (per node) to reserve",
-	"arc.postgres.server.group.cpu.max": "Max CPU cores (per node) to allow",
-	"arc.postgres.server.group.memory.min": "Min memory MB (per node) to reserve",
-	"arc.postgres.server.group.memory.max": "Max memory MB (per node) to allow",
-	"arc.postgres.server.group.backup.classes": "Backup volume storage classes",
-	"arc.postgres.server.group.backup.classes.description": "A comma-separated list of existing Kubernetes storage classes to use for the PostgreSQL server group's backup volumes, one per backup tier. If provided, backup volume sizes must also be provided.",
-	"arc.postgres.server.group.backup.sizes": "Backup volume sizes (MB)",
-	"arc.postgres.server.group.backup.sizes.description": "A comma-separated list of the number of megabytes (per node) that will be requested for the PostgreSQL server group's backup volumes, one per backup tier. If specified, backups will be enabled. In this configuration a separate backup volume is used for each node.",
-	"arc.postgres.server.group.backup.claims": "Backup volume claims",
-	"arc.postgres.server.group.backup.claims.description": "A comma-separated list of existing Kubernetes persistent volume claims (in the same namespace) to use for the PostgreSQL server group's backups, one per backup tier. If specified, backups will be enabled. In this configuration the backup volumes are shared across all nodes.",
-	"arc.postgres.server.group.backup.full.interval": "Minutes between full backups",
-	"arc.postgres.server.group.backup.delta.interval": "Minutes between delta backups",
-	"arc.postgres.server.group.backup.retention.min": "Minimum trim settings",
-	"arc.postgres.server.group.backup.retention.min.description": "A list of trim settings that specifies the minimum number of days/size/counts of backups to preserve per tier. Each trim setting contains 1 or more trim values separated by commas and each tier is separated by a semicolon. Possible trim values include '7d', '10GB', or '50'.",
-	"arc.postgres.server.group.backup.retention.min.validation.description": "Minimum trim settings must contain 1 or more trim values separated by commas with each tier separated by a semicolon.",
-	"arc.postgres.server.group.backup.retention.max": "Maximum trim settings",
-	"arc.postgres.server.group.backup.retention.max.description": "A list of trim settings that specifies the maximum number of days/size/counts of backups to preserve per tier. Each trim setting contains 1 or more trim values separated by commas and each tier is separated by a semicolon. Possible trim values include '7d', '10GB', or '50'.",
-	"arc.postgres.server.group.backup.retention.max.validation.description": "Maximum trim settings must contain 1 or more trim values separated by commas with each tier separated by a semicolon.",
+	"arc.postgres.server.group.cores.request": "Min CPU cores (per node) to reserve",
```

Comment:
@bergeron Can you check through the strings to make sure things still make sense?